doc/deparam.md

```diff
@@ -0,0 +1,67 @@
+@function can-route.deparam deparam
+@parent can-route.static
+
+@description Extract data from a route path.
+
+@signature `route.deparam(url)`
+
+  Extract data from a url, creating an object representing its values.
```

Comment:
We might want to clarify that it doesn't come from the whole url.  At least I don't think that something like `deparam(window.location.href)` works.  We might be able to give people an easy way of doing this though.  Ping me and we'll talk about it.

Reply:
It doesn't work, just returning `undefined`.
I added suggestions of what kind of fragments this would work well on like location.hash or location.search